### PR TITLE
Mr. Wang wanted wrong code

### PR DESCRIPTION
**Pull request by bossy64**
@twang Mr wang wanted bad code

### CODE REVIEW

**Comment by DolanOats** [comment]
This is the corrected code:
public class Egg2 {
	public static void bob(){
        System.out.println("  _______  ");
        System.out.println(" /       \\ ");
        System.out.println("/         \\");
        System.out.println("\\         /");
        System.out.println(" \\_______/ ");
        **System.out.println("-\"-'-\"-'-\"-");**
    }
    public static void bbob(){
        System.out.println("\\         /");
        System.out.println(" \\_______/ ");
    }
    public static void bbbob(){
         System.out.println("  _______  ");
        System.out.println(" /       \\ ");
       System.out.println("/         \\");
            **System.out.println("-\"-'-\"-'-\"-");**
    }
    public static void main(String[] args){
        bob();
        bob();
        bbob();
        bbbob();
        bbob();
    }
}

The bolded areas are where it is wrong. You originally had ("-\"-'-\"-'-\"-\") for both of them. What happened to the code was that  because you put a slash in front of the last quotation, it reprinted it and the quotation was never closed. When you get rid of the slash, the quotation is closed and you have a println statement that works.

File: BadCode.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+public class Egg2 {
+	public static void bob(){
+        System.out.println("  _______  ");
+        System.out.println(" /       \\ ");
+        System.out.println("/         \\");
+        System.out.println("\\         /");
+        System.out.println(" \\_______/ ");
+        System.out.println("-\"-'-\"-'-\"-\");
```
Comment (DolanOats) [inline BadCode.java]: System.out.println("-\"-'-\"-'-\"-\"); you want it to look like System.out.println("-\"-'-\"-'-\"-");

File: BadCode.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    
+    public static void bbob(){
+        System.out.println("\\         /");
+        System.out.println(" \\_______/ ");
+    }
+    public static void bbbob(){
+         System.out.println("  _______  ");
+        System.out.println(" /       \\ ");
+        System.out.println("/         \\");
+            System.out.println("-\"-'-\"-'-\"-\");
```
Comment (DolanOats) [inline BadCode.java]: Same thing here

**Comment by twang** [comment]
Thanks @bossy64 and @DolanOats!

**Pull request closed without merging**